stdlib/builtins.pyi

```diff
@@ -324,12 +327,12 @@ class complex:
     def __add__(self, __x: complex) -> complex: ...
     def __sub__(self, __x: complex) -> complex: ...
     def __mul__(self, __x: complex) -> complex: ...
-    def __pow__(self, __x: complex, mod: None = ...) -> complex: ...
+    def __pow__(self, __x: complex | float, mod: None = ...) -> complex: ...
```

Comment:
This change is not needed. PEP 484 says complex duck types for complex, float and int

Reply:
Thanks!